Sources/KSCrashReportingCore/KSHTTPMultipartPostBody.m

```diff
@@ -26,8 +26,21 @@
 
 #import "KSHTTPMultipartPostBody.h"
 
-#import "NSMutableData+AppendUTF8.h"
-#import "NSString+URLEncode.h"
+static void appendUTF8String(NSMutableData *data, NSString *string)
```

Comment:
It became less OOP)

Reply:
Yeah, but at the same time categories are not really OOP :)